packages/client/components/EditGitHubLabelTemplateModal.tsx

```diff
@@ -0,0 +1,108 @@
+import styled from '@emotion/styled'
+import React from 'react'
+import {PALETTE} from '~/styles/paletteV3'
+import useForm from '../hooks/useForm'
+import DialogContainer from './DialogContainer'
+import DialogContent from './DialogContent'
+import DialogTitle from './DialogTitle'
+import BasicInput from './InputField/BasicInput'
+import PlainButton from './PlainButton/PlainButton'
+import RaisedButton from './RaisedButton'
+import SecondaryButton from './SecondaryButton'
+
+interface Props {
+  closePortal: () => void
+  updateLabelTemplate: (labelTemplate: string) => () => void
+  defaultValue: string
+  placeholder: string
+}
+
+const StyledDialogContainer = styled(DialogContainer)({
+  width: 480
+})
+
+const ButtonGroup = styled('div')({
+  marginTop: '24px',
+  display: 'flex',
+  justifyContent: 'flex-end'
+})
+
+const StyledTip = styled('span')({
+  fontSize: 16,
+  margin: 0,
+  padding: 4
+})
+
+const WildcardTip = styled(StyledTip)({
+  background: PALETTE.SLATE_300,
+  borderRadius: 8,
+  fontWeight: 600
+})
+const StyledRaisedButton = styled(RaisedButton)({
+  marginLeft: 16
+})
+
+const LabelTemplateInput = styled(BasicInput)({
+  color: PALETTE.SLATE_700,
+  fontSize: 16,
+  marginBottom: 8,
+  outline: 0,
+  backgroundColor: 'transparent',
+  width: '100%'
+})
+
+const EditGitHubLabelTemplateModal = (props: Props) => {
+  const {closePortal, defaultValue, placeholder, updateLabelTemplate} = props
+  const INPUT_NAME = 'labelTemplate'
+  console.log({defaultValue})
+  const {fields, onChange, setValue} = useForm({
+    [INPUT_NAME]: {
+      getDefault: () => defaultValue
+    }
+  })
+  const labelTemplateField = fields[INPUT_NAME]
+  const {value} = labelTemplateField
+  const onSave = () => {
```

Comment:
It might just be me, but after saving the new GitHub label, I went to GitHub to see the new label and was surprised that it wasn't already there. Then I returned to Parabol and noticed the `Update` button. 

I think the `Update` button works well when updating the final story points as it's right next to the input, and we couldn't submit the value on each keystroke. Still, I wonder whether saving the new label or selecting `As a label`, `As a comment`, or `Do Not Update` could submit immediately without needing to clicking `Update`?

Not a show stopper by any means, just curious to get your thoughts

Reply:
that's a real good idea you've got. basically selecting a menu item should also trigger a submit if there is a pendingScore?
if we do it for GH we should do it for Jira, too, & i think that works OK.

So the possible gotcha with the current pattern is: if I change the field, I have to click "Update" again
The gotcha with the future pattern is: the menu item select behavior is conditional (push vs. no push) based on the value of the pending score (non-null vs null). If i click the wrong menu item, there will be a consequence.

So, what's the probability that someone will click a menu item & NOT hit update? probably pretty darn low. We should implement your strategy :+1: